### DISCUSSION TITLE
Country field

### DISCUSSION THREAD

**Discussion by kevin-vanb** [liedekef/events-made-easy#469] [General]
I have a list of countries, members indicated their country during registration or this was done in the backend, but recently we saw that country was emptied, even after reselecting and updating the corresponding person info, the field remains empty.

Any idea?

**Comment by liedekef** [comment]
This is not reproducable here: if I select a country for a person, it stays correctly selected.
Which version of EME? And what is the definition of the country (did you have a language set for it)?

**Reply by kevin-vanb** [reply]
![image](https://github.com/liedekef/events-made-easy/assets/123742526/53aa77e6-17ed-4c2f-a1a7-1712990050db)
EME Version 2.4.32
When I select the French version of Belgium it seems to work or am I doing something wrong with the countries like this?

**Reply by liedekef** [reply]
You can't have 2 definitions of the same Country and leave the language empty for both ... that will cause problems.

**Reply by kevin-vanb** [reply]
Ok, thanks for the clarification :-)

**Reply by liedekef** [reply]
The next version will prevent this from happening again, but currently this means you added België or Belgique just before the problem started :-)
